server/src/ito.proto

```diff
@@ -207,8 +207,18 @@ message DeleteUserDataRequest {
 
 // Advanced Settings
 // -----------------------------------------------------------------
+
 message LlmSettings {
   string asr_model = 1;
+  string asr_provider = 2;
+  string asr_prompt = 3;
+  string llm_provider = 4;
+  string llm_model = 5;
+  float llm_temperature = 6;
+  string transcription_prompt = 7;
+  string editing_prompt = 8;
```

Comment:
Should we have a prompt field and then a mode field? Otherwise we'll need to add prompts for every single mode

Reply:
Nevermind, I thought this was a message being passed with the transcription request. This is the message for updating settings, in which case yes we'll need all the prompts for all the modes